### PR TITLE
enh(release): Add Centreon PPM 21.04.1 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x (master)

### CODE REVIEW

File: fr/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -141,6 +141,17 @@ Release date: `9 novembre 2021`
 
 ## Centreon Plugin Packs Manager
 
+### 21.04.2
```
Comment (mdebauchez) [inline fr/releases/centreon-commercial-extensions.md:144]: ### 21.04.1

File: en/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -142,6 +142,17 @@ Release date: `November 9, 2021`
 
 ## Centreon Plugin Packs Manager
 
+### 21.04.2
```
Comment (mdebauchez) [inline en/releases/centreon-commercial-extensions.md:145]: ### 21.04.1

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

File: en/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -142,6 +142,17 @@ Release date: `November 9, 2021`
 
 ## Centreon Plugin Packs Manager
 
+### 21.04.1
+
```
Comment (chgautier) [inline en/releases/centreon-commercial-extensions.md:146]:
```suggestion

`Release date: December 10, 2021`

```

File: fr/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -141,6 +141,17 @@ Release date: `9 novembre 2021`
 
 ## Centreon Plugin Packs Manager
 
+### 21.04.1
+
```
Comment (chgautier) [inline fr/releases/centreon-commercial-extensions.md:145]:
```suggestion

`Release date: December 10, 2021`

```

**Review by chgautier — CHANGES_REQUESTED** [review]

**Review by chgautier — APPROVED** [review]

**Pull request merged**